lib/enhancements/edition.rb

```diff
@@ -43,6 +43,8 @@ def self.state_names
     )
   }
 
+  scope :published, -> { where(state: "published").desc(:created_at) }
```

Comment:
I think what might avoid adding a duplicate scope to the `Edition` model might be something like:
```ruby
Edition.published.order(created_at: :desc)
```
as @tuzz says, use the existing `published` scope and modify it by chaining your own order clause to it.